validator/src/main/java/com/amazon/aoc/clients/ExpectedResultClient.java

```diff
@@ -0,0 +1,51 @@
+package com.amazon.aoc.clients;
+
+import com.amazon.aoc.models.Context;
+import com.amazon.aoc.models.prometheus.PrometheusMetric;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+
+import java.io.IOException;
+import java.util.List;
+
+public class ExpectedResultClient {
```

Comment:
the name of "ExpectedResultClient" made me think it's a generic client for all the types of metrics including cw and prometheus but looks like it's just for prometheus?